site/app/controllers/PollController.php

```diff
@@ -341,7 +364,7 @@ public function submitEdits(): RedirectResponse {
             $this->core->addErrorMessage("Invalid Poll ID");
             return new RedirectResponse($returnUrl);
         }
-        if (!isset($_POST["response_count"]) || !isset($_POST["name"]) || !isset($_POST["question"]) || !isset($_POST["release_date"])) {
+        if (!isset($_POST["response_count"]) || !isset($_POST["name"]) || !isset($_POST["question"]) || !isset($_POST["question_type"]) || !isset($_POST["release_date"])) {
```

Comment:
can do a similar foreach here. Perhaps define the expected fields as a class property to iterate over so it's only defined in one place.